### PR TITLE
Create PacBio pblaa recipe.

### PR DESCRIPTION
**Pull request by ylipacbio**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/pblaa/2.4.2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+{% set version = "2.4.2" %}
+{% set laa_sha256 = "563a2f68c370f9e4f6eb3ac6656ee8a37be3abdae64124d7d077488dcb332172" %}
+{% set laagc_sha256 = "ec366dcbf1c6197e9baf4cb5147c8b41d5d76d860ed835ae6407c88d870205e0" %}
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source: # must keep [linux] to prevent it being installed on mac os
```
Comment (bgruening) [inline recipes/pblaa/2.4.2/meta.yaml]: You can skip a build on OSX with: `skip: True  # [osx]` in the build/skip section.

Comment (ylipacbio) [reply recipes/pblaa/2.4.2/meta.yaml]: Thanks for the reminder. I have added `skip: True # [osx]` in the build/skip section.  I will remove this comment.

File: recipes/pblaa/2.4.2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  version: {{ version }}
+
+source:
+  - url: https://github.com/PacificBiosciences/pblaa/releases/download/v{{ version }}/laa # [linux]
+    sha256: {{ laa_sha256 }} # [linux]
+  - url: https://github.com/PacificBiosciences/pblaa/releases/download/v{{ version }}/laagc # [linux]
+    sha256: {{ laagc_sha256 }} # [linux]
+
+about:
+  home: https://github.com/ylipacbio/pblaa
```
Comment (pb-dseifert) [inline recipes/pblaa/2.4.2/meta.yaml]: not the official PacBio link

File: recipes/pblaa/2.4.2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  recipe-maintainers:
+    - ylipacbio
+    - armintoepfer
+    - pb-dseifert
+    - mhsieh
+
+build:
+  number: 0
+  skip: True # [osx]
+
```
Comment (pb-dseifert) [inline recipes/pblaa/2.4.2/meta.yaml]: spurious empty line

Comment (ylipacbio) [reply recipes/pblaa/2.4.2/meta.yaml]: Thanks for the review, I have removed spurious empty lines from this PR and squashed the commits.

**Pull request merged**